pkg/edit/complete/complete_test.go

```diff
@@ -280,10 +279,10 @@ func cb(s string) CodeBuffer { return CodeBuffer{s, len(s)} }
 func c(s string) completion.Item { return completion.Item{ToShow: s, ToInsert: s} }
 
 func fc(s, suffix string) completion.Item {
-	return completion.Item{ToShow: s, ToInsert: s + suffix,
+	return completion.Item{ToShow: s, ToInsert: parse.Quote(s) + suffix,
 		ShowStyle: ui.StyleFromSGR(lscolors.GetColorist().GetStyle(s))}
 }
 
 func r(i, j int) diag.Ranging { return diag.Ranging{From: i, To: j} }
 
-func withPathSeparator(d string) string { return d + parse.Quote(string(os.PathSeparator)) }
+func withPathSeparator(d string) string { return d + string(os.PathSeparator) }
```

Comment:
`withPathSeparator(d)` is not any shorter than `d + pathSeparator`. I'd prefer that you remove this function.

Reply:
Agree ;)